### PR TITLE
feat(*): update CHANGELOG.md file

### PR DESCRIPTION
**Pull request by linhbkhn95**
### Why did we need it?
- Update the `CHANGELOG.md` file to prepare the release.

### CODE REVIEW

**Comment by linhbkhn95** [comment]
@hibiken @kamikazechaser I just updated the `CHANGELOG.md` file. So, please feel free to look it over and help me.

**Review by kamikazechaser — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.25.0] - 2023-07-30
```
Comment (hibiken) [inline CHANGELOG.md:10]: don't add this line yet, we are not releasing this version yet. You can add your entry under `[Unreleased]` section (just simply drop this line).

**Review by hibiken — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (linhbkhn95) [reply CHANGELOG.md]: yeap, I got it. I just updated it.

**Comment by linhbkhn95** [comment]
@kamikazechaser @hibiken anything else about this PR?

**Comment by hibiken** [comment]
Sorry for the delay here, would you mind rebase/resolve conflicts? I'll merge after that

**Comment by linhbkhn95** [comment]
Sorry, Can u check this PR?
https://github.com/hibiken/asynq/pull/758
Please Merged it first, and the current PR.
Thanks @hibiken

**Review by hibiken — APPROVED** [review]

**Comment by kamikazechaser** [comment]
I'll cherry-pick /adc5d9b8ee6577ae56a590d819b131b47d2e105e

**Pull request merged**